src/pages/Transaction/TransactionComp/NodeTransactionOverview.tsx

```diff
@@ -0,0 +1,247 @@
+import { shannonToCkb, useFormatConfirmation } from '../../../utils/util'
+import { getTransactionOutputCells, calculateFeeByTxIO, checkIsCellBase } from '../../../utils/transaction'
+import { TransactionBlockHeightPanel, TransactionOverviewPanel } from './styled'
+import { useLatestBlockNumber } from '../../../services/ExplorerService'
+import { NodeRpc } from '../../../services/NodeService'
+import { Card, CardCellInfo, CardCellsLayout, HashCardHeader } from '../../../components/Card'
+import RawTransactionView from '../../../components/RawTransactionView'
+import { ReactComponent as DownloadIcon } from './download.svg'
+import { useIsMobile } from '../../../hooks'
+import { useCKBNode } from '../../../hooks/useCKBNode'
+import styles from './TransactionOverview.module.scss'
+import TransactionParameters from '../../../components/TransactionParameters'
+
+const showTxStatus = (txStatus: string) => txStatus?.replace(/^\S/, s => s.toUpperCase()) ?? '-'
+const TransactionBlockHeight = ({ txStatus }: { txStatus: NodeRpc.TransactionWithStatus['tx_status'] }) => (
+  <TransactionBlockHeightPanel>
+    {txStatus.status === 'committed' ? (
+      <Link to={`/block/${parseInt(txStatus.block_number, 16)}`}>{localeNumberString(txStatus.block_number)}</Link>
+    ) : (
+      <span>{showTxStatus(txStatus.status)}</span>
+    )}
+  </TransactionBlockHeightPanel>
+)
+
+export const NodeTransactionOverviewCard: FC<{
+  transactionWithStatus: NodeRpc.TransactionWithStatus
+}> = ({ transactionWithStatus }) => {
+  const { transaction: rawTransaction, tx_status: txStatus, cycles } = transactionWithStatus
+  const { nodeService } = useCKBNode()
+  const { t } = useTranslation()
+  const isMobile = useIsMobile()
+  const tipBlockNumber = useLatestBlockNumber()
+  const [detailTab, setDetailTab] = useState<'params' | 'raw' | null>(null)
+
+  const header = useQuery(
+    ['node', 'header', txStatus.status === 'committed' ? txStatus.block_number : null],
+    () => (txStatus.status === 'committed' ? nodeService.rpc.getHeaderByNumber(txStatus.block_number) : null),
+    {
+      staleTime: Infinity,
+      enabled: txStatus.status === 'committed',
+    },
+  )
+
+  const inputCells = useQuery(
+    ['node', 'inputCells', rawTransaction?.hash],
+    () => (rawTransaction ? nodeService.getInputCells(ResultFormatter.toTransaction(rawTransaction).inputs) : []),
+    {
+      enabled: Boolean(rawTransaction),
+    },
+  )
+  const confirmation = (() => {
+    if (tipBlockNumber && txStatus.status === 'committed' && txStatus.block_number) {
+      return Number(tipBlockNumber) - Number(txStatus.block_number)
+    }
+
+    return 0
+  })()
+  const formatConfirmation = useFormatConfirmation()
+
+  if (!rawTransaction) {
+    return null
+  }
+
+  const tx = ResultFormatter.toTransaction(rawTransaction)
+  const isCellBase = checkIsCellBase(tx)
+  // eslint-disable-next-line no-underscore-dangle
+  const txSize = common.__tests__.getTransactionSizeByTx(tx)
```

Comment:
If `common.__tests__` is inevitable, please wrap it in a function to keep all dirty work in the standalone function.